apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -127,3 +131,13 @@ export default function PokemonLink({
   );
 }
 ```
+
+import NavigationButtons from '../../../components/navigation-buttons'
+
+<NavigationButtons
+  prev={{ title: 'API fetching', href: '/documentation/tutorial/api-fetching' }}
+  next={{
+    title: 'Dynamic routes',
+    href: '/documentation/tutorial/dynamic-routes',
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space before the closing parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `}`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChBkYTI3ODFjOGIwNTAxNjc0EgxMYW5ndWFnZVRvb2w= -->